### ISSUE TITLE
Batchsize Issue

### ISSUE THREAD

**Issue by we1sher** [VISION-SJTU/RSD#2]
Your method is conducted on 2 GPUs and its batchsize is 128*2=256 for CNNs on Cifar-100, but OFA is conducted on 8 GPUs whose batchsize is 128*8=1024, which results in a acc gap. How do you solve this issue?

**Comment by spacewanderx** [comment]
Hi, thanks for your interest in our work.

Due to limited resources we choose to go with 2 GPUs for CIFAR experiments. We made efforts to run OFA with our same settings where possible, and confirmed that performance-wise the edge remains. 

For example,  for Swin-T->R18, our logs record a 81.11 reproduced OFA acc. (v.s. 80.54 in paper v.s. 83.92 for RSD). So it shouldn't make a big difference.

Hope it helps.

**Comment by we1sher** [comment]
Hi, thanks for your response.

My issue is not about the experimental results of Transformer->CNN, but about the results of CNN->Transformer on CIFAR-100. For 8GPUs, convnext-t & swin-p can produce 78%~79% Acc, while for 2GPUs, its performance is improved to 82%~83%.

**Comment by we1sher** [comment]
And if possible, could you please disclose how many GPUs you used to train the models on Imagenet-1k?

**Comment by spacewanderx** [comment]
Hi. For 1), we believe that fluctuation in results is expected, and the learning rate may need to be tuned when changing the GPU configuration. We did not experiment with 8-GPU training, but your information is valuable for reproducibility. For future experiments, it'd be great to reproduce and present existing methods using the same GPU configs. For 2), we used 2-4 GPUs for ImageNet.

**Comment by we1sher** [comment]
> Hi. For 1), we believe that fluctuation in results is expected, and the learning rate may need to be tuned when changing the GPU configuration. We did not experiment with 8-GPU training, but your information is valuable for reproducibility. For future experiments, it'd be great to reproduce and present existing methods using the same GPU configs. For 2), we used 2-4 GPUs for ImageNet.

Really appreciate for your answers!!
